youToBe.css

```diff
@@ -0,0 +1,66 @@
+body {
+    background-image: url(https://images.saymedia-content.com/.image/t_share/MTc4NzM1OTc4MzE0MzQzOTM1/how-to-create-cool-website-backgrounds-the-ultimate-guide.png);
+    background-position: center;
+    background-size: cover;
+    background-repeat: round;
+}
+
+#homeBody {
+    width: 900px;
+    margin: auto;
+    background-color: azure;
+    font-family: Verdana, Geneva, Tahoma, sans-serif;
+}
+
+header {
+    margin: auto;
+    text-align: center;
+    font-size: 30px;
+    padding: 25px;
+}
+
+nav {
+    background-color: #960D85;
+    list-style-type: none;
+    text-align: center;
+    margin: 0;
+    padding: 0;
```

Comment:
I kindly recommend using the _display_ property to make you menu and your site more flexible so that your great content can be seen the best way possible on any screen.

![image](https://user-images.githubusercontent.com/96145777/220154986-517972af-3c47-4192-a668-41e841662271.png)